libs/gi/db/src/Database/DataManagers/ArtifactDataManager.ts

```diff
@@ -325,6 +325,7 @@ export function cachedArtifact(
   id: string
 ): { artifact: ICachedArtifact; errors: string[] } {
   const { location, lock, setKey, slotKey, rarity, mainStatKey } = flex
+  let { totalRolls } = flex
```

Comment:
Don't we need to pass this into the substat calculation portion at 360? So the calculation knows how many total rolls to allocate, instead of guessing between low/high